code/modules/mining/laborcamp/laborshuttle.dm

```diff
@@ -14,8 +14,8 @@
 	circuit = /obj/item/circuitboard/labor_shuttle/one_way
 	req_access = list( )
 
-/obj/machinery/computer/shuttle/labor/one_way/Topic(href, href_list)
-	if(href_list["move"])
+/obj/machinery/computer/shuttle/labor/one_way/tgui_act(action, list/params, datum/tgui/ui, datum/tgui_state/state)
```

Comment:
`list/` can be omitted from `list/params` and the last 2 arguments are typically not added to the signature because there are almost no cases they'll be used.